packages/nvidia_nat_rag/src/nat/plugins/rag/rag_functions.py

```
@@ -0,0 +1,213 @@
+                vdb_top_k=config.vdb_top_k,
+                embedding_endpoint=config.embedding_endpoint,
+            )
+
+            if not citations or not hasattr(citations, "results") or not citations.results:
+                return "No documents found for the given query."
+
+            # Format the results
+            results = []
+            for idx, citation in enumerate(citations.results):
+                doc_name = getattr(citation, "document_name", f"Document {idx + 1}")
+                content = getattr(citation, "content", "")
```

Ruff (0.14.11)

Local variable `content` is assigned to but never used

Remove assignment to unused variable `content`

(F841)